.vib/parse/goss/vars.yaml

```diff
@@ -0,0 +1,18 @@
+binaries:
+  - node
+  - parse-server
```

Comment:
I would maybe check the permission changes done at the postunpack stage to parse binaries affect this `parse-server`, wdyt?

Reply:
I don't think it is needed. If not, `which parse-server` would not return any result. I think it is enough to ensure that it is executable, which is achievable with the `which` command (and in the end is what we are trying to achieve with the permissions change).